lib/challenge.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+let milli = 1000000;
```

Comment:
This is intended to be a constant, right? In that case, you should make it a constant.